src/gui/codeeditors/qgscodeeditorpython.h

```diff
@@ -120,6 +122,13 @@ class GUI_EXPORT QgsCodeEditorPython : public QgsCodeEditor
 
     bool checkSyntax() override;
 
+    /**
+     *  Checks  whether QWebEngineView is available to display context help
+     *
+     * \since QGIS 3.42
+     */
+    static bool hasWebEngine();
```

Comment:
Can you move this to QgsGui ? It's not specifically related to QgsCodeEditorPython, and I think having it in a more generic place would be valuable.